### ISSUE TITLE
Linux Mint theme window not opening after mint update 19.2

### ISSUE THREAD

**Issue by Dhineshss** [linuxmint/cinnamon#8761]
I upgraded linux mint from 19.1 to 19.2. After the update, theme window is not opening. Nothing happens when click the theme icon in the settings.

![Screenshot from 2019-08-03 13-08-04](https://user-images.githubusercontent.com/6842173/62409082-cba65e80-b5ef-11e9-80af-f1d0c8b7c257.png)

**Comment by mtwebster** [comment]
Can you share the contents of your `~/.config/gtk-3.0/gtk.css` file?  (If you have one)

Thanks

**Comment by claudiux** [comment]
@Dhineshss Try to reinstall the cinnamon-common package:
`sudo apt-get install --reinstall cinnamon-common`

**Comment by claudiux** [comment]
@mtwebster I don't have any `~/.config/gtk-3.0/gtk.css` file and cinnamon-settings works fine.
@Dhineshss Maybe trying to rename this file, if it exists:
mv ~/.config/gtk-3.0/gtk.css ~/.config/gtk-3.0/gtk.css_OLD
Then, log off and log on you.

**Comment by Dhineshss** [comment]
@mtwebster gtk.css file contains the followings
```
@import url('/home/dhinesh/.local/share/gnome-shell/extensions/no-title-bar@franglais125.gmail.com/stylesheet.css');
@import url('/home/dhinesh/.local/share/gnome-shell/extensions/no-title-bar@franglais125.gmail.com/stylesheet-tiled.css');
```
I just checked the url files are exists in the system.

**Comment by Dhineshss** [comment]
@claudiux  Thanks your solutions works. I just renamed the file 'gtk.css' & works immeadiatly. Thank you

**Comment by claudiux** [comment]
@Dhineshss The `~/.local/share/gnome-shell` doesn't exist in my home directory. Maybe a filth of an old installation?

**Comment by claudiux** [comment]
@Dhineshss Maybe you can close this issue?

**Comment by cosminribo** [comment]
This is an old and closed issue but the solution of moving the `gtk.css` file out of the way solved the problem for me on a Ubuntu 18.04 running Cinnamon 4.2.4. Thank you.
